api/third_party/openapi/odpf/optimus/plugins/v1beta1/dependency_resolver.swagger.json

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
Until here, all files should not be added.

Reply:
done